.github/MAPS_AND_AWAY_MISSIONS.md

```diff
@@ -1,13 +1,26 @@
 ## MAPS
 
-/tg/station currently comes equipped with five maps.
+/tg/station currently has in rotation five maps.
 
-* [BoxStation (default)](https://tgstation13.org/wiki/Boxstation)
 * [MetaStation](https://tgstation13.org/wiki/MetaStation)
 * [DeltaStation](https://tgstation13.org/wiki/DeltaStation)
+* [IceboxStation](https://tgstation13.org/wiki/IceboxStation)
+* [KiloStation](https://tgstation13.org/wiki/KiloStation)
+* [TramStation](https://tgstation13.org/wiki/Tramstation)
+
+Outdated, removed or not in rotation maps.
+* [BoxStation](https://tgstation13.org/wiki/Boxstation)
 * [PubbyStation](https://tgstation13.org/wiki/PubbyStation)
 * [DonutStation](https://tgstation13.org/wiki/Donutstation)
-
+* [AsteroidStation](https://tgstation13.org/wiki/AsteroidStation)
+* [BirdboatStation](https://tgstation13.org/wiki/BirdboatStation)
+* [CereStation](https://tgstation13.org/wiki/CereStation)
+* [DiscStation](https://tgstation13.org/wiki/Discstation)
+* [DreamStation](https://tgstation13.org/wiki/Dreamstation)
+* [EfficiencyStation](https://tgstation13.org/wiki/EfficiencyStation)
+* [MiniStation](https://tgstation13.org/wiki/MiniStation)
+* [OmegaStation](https://tgstation13.org/wiki/OmegaStation)
+* [RuntimeStation(Debugging)](https://tgstation13.org/wiki/RuntimeStation)
```

Comment:
I mean, it's not in the main rotation but it's technically still in the game, this is a mite misleading

Reply:
`Outdated, removed or not in rotation maps.` why misleading?

Reply:
Cause every one of these was removed outside of runtime. Maybe give it its own section as a debug map alongside the multiz one?

Reply:
aye aye